plugins/tracker-resources/src/components/issues/edit/EditIssue.svelte

```diff
@@ -276,8 +283,22 @@
     </svelte:fragment>
 
     {#if hasParentIssue}
-      <div class="mb-6">
+      <div class="mb-6 flex-row-center">
         <SubIssueSelector {issue} />
+        {#if hasParentIssue}
```

Comment:
Extra check, we have the same in 285

Reply:
Maybe we should hide or disable button if readonly?

Reply:
Thanks, Fixed.